crypto/compactcert/builder.go

```diff
@@ -41,12 +40,16 @@ type sigslot struct {
 // Builder keeps track of signatures on a message and eventually produces
 // a compact certificate for that message.
 type Builder struct {
-	Params
-	sigs          []sigslot // Indexed by pos in participants
-	sigsHasValidL bool      // The L values in sigs are consistent with weights
-	signedWeight  uint64    // Total weight of signatures so far
-	participants  []basics.Participant
-	parttree      *merklearray.Tree
+	data           StateProofMessageHash
+	round          uint64
+	sigs           []sigslot // Indexed by pos in participants
+	sigsHasValidL  bool      // The L values in sigs are consistent with weights
```

Comment:
`sigsHasValidL` is redundant and can be removed.
It is only used in `coinIndex`, which is a local function. `coinIndex` is only called from `Build`, after setting `sigsHasValidL` to `true`.